goreman_test.go

```diff
@@ -33,6 +33,7 @@ func startGoreman(ctx context.Context, t *testing.T, ch <-chan os.Signal, file [
 }
 
 func TestGoreman(t *testing.T) {
+	defer wg.Wait()
```

Comment:
please remove this `defer wg.Wait()` at all.

Reply:
Done!